src/client/components/ConfirmJoinOrganization.tsx

```diff
@@ -80,16 +80,18 @@ const ConfirmJoinOrganization = ({
       </Box>
       <Flex sx={{ flexDirection: "column" }}>
         <Box>
```

Comment:
```suggestion
        <Box sx={{ marginBottom: 2 }}>
```

@tgilcrest I think this looks a little better with a bit of spacing between the two boxes:

With spacing:
![image](https://user-images.githubusercontent.com/66973361/118035043-b6b83c80-b338-11eb-937b-900c5c3d94dc.png)

Without:
![image](https://user-images.githubusercontent.com/66973361/118035181-e23b2700-b338-11eb-91c1-e3172443c07b.png)

Reply:
Looks great!